src/viam/sdk/components/service_base.cpp

```diff
@@ -1,5 +1,9 @@
 #include <viam/sdk/components/service_base.hpp>
 
 namespace viam {
-namespace sdk {}
+namespace sdk {
+const ResourceManager& ComponentServiceBase::manager() const {
```

Comment:
@acmorrow if I understand correctly, this `const` is still meaningful since it prevents the underlying data being referred from being changed. Is this correct?